scitt/check_operation_status.py

```diff
@@ -91,11 +104,22 @@ def main():
         default=default_token_file_name,
     )
 
+    # log level
+    parser.add_argument(
+        "--log-level",
+        type=str,
+        help="log level. for any individual poll errors use DEBUG, defaults to WARNING",
+        default="WARNING",
+    )
+
     args = parser.parse_args()
 
```

Comment:
Why not have a run() method that main calls then do the usual idiomatic sinlge line exception printing for example,

```
try:
    # Simulate an error
    1 / 0
except Exception as e:
    # Format the traceback as a single line
    tb_str = ''.join(traceback.format_exception(etype=type(e), value=e, tb=e.__traceback__)).replace('\n', ' ')
    # Print to stderr
    print(tb_str, file=sys.stderr)
```

Reply:
It is not unusual to exose the raw traceback when the user has explicitly set loglevel DEBUG. the loggin package allows this to be introspected I beleive

Reply:
have handled the timeout exception within main(), so now we don't print the full stack trace, just the correct timeout message and exit with exit code 1.

Also reduced the timeout duration to 60 seconds from 360 seconds (6 minutes) as a merklelog world is much faster :)